src/dropdown/list/dropdown-list.component.ts

```diff
@@ -361,6 +361,9 @@ export class DropdownList implements AbstractDropdownView, AfterViewInit, OnDest
 	getSelected(): ListItem[] {
 		let selected = this.getListItems().filter(item => item.selected);
 		if (selected.length === 0) {
+			if (this.type === "multi") {
+				return [];
+			}
 			return null;
```

Comment:
Instead of special casing this return, would it make sense to return `[]` in all cases? What else in the component would that affect?

Reply:
If we return [] in all cases, it gives the following error if the dropdown is not of type 'multi':

```
VM928 Dropdown.ngfactory.js:109 ERROR TypeError: Cannot read property 'content' of undefined
    at Dropdown.webpackHotUpdate../src/dropdown/dropdown.component.ts.Dropdown.getDisplayStringValue (dropdown.component.ts:483)
    at Object.eval [as updateRenderer] (VM710 Dropdown.ngfactory.js:113)
    at Object.debugUpdateRenderer [as updateRenderer] (core.js:11088)
    at checkAndUpdateView (core.js:10464)
    at callViewAction (core.js:10700)
    at execEmbeddedViewsAction (core.js:10663)
    at checkAndUpdateView (core.js:10460)
    at callViewAction (core.js:10700)
    at execComponentViewsAction (core.js:10642)
    at checkAndUpdateView (core.js:10465)
```

Reply:
Let's fix those issues where the come up, and return `[]` for all cases ... I don't expect this to be a breaking change in practice

Reply:
Fixed!